bee-protocol/src/worker/solidifier/milestone.rs

```diff
@@ -84,26 +85,28 @@ impl MilestoneSolidifierWorker {
 
         // TODO this may request unpublished milestones
         for index in solid_milestone_index..solid_milestone_index + MILESTONE_REQUEST_RANGE as u32 {
-            if !tangle().contains_milestone(index.into()) {
+            let index = index.into();
+            if !tangle().contains_milestone(index) {
                 Protocol::request_milestone(index, None);
             }
         }
     }
 
     async fn solidify_milestone(&self) {
-        let target_index = *tangle().get_solid_milestone_index() + 1;
+        let target_index = tangle().get_solid_milestone_index() + 1.into();
```

Comment:
Can we use `MilestoneIndex(1)` instead ? `1.into()` is weird

Reply:
Generally I'd tend to use the constructor more often than into, IMO this is more readable. Even more in this case.

Reply:
Definitly agree with you on this! Changed.